_inc/client/plans/single-product-search/index.jsx

```diff
@@ -45,17 +46,12 @@ function getTierLabel( priceTierSlug, recordCount ) {
 	}
 }
 
-function handleSelectedTimeframeChangeFactory( setTimeframe ) {
-	return event => setTimeframe( event.target.value );
-}
-
 export function SingleProductSearchCard( props ) {
-	const [ timeframe, setTimeframe ] = useState( 'yearly' );
```

Comment:
We'll want to remove the unused `useState` import with this change.

Reply:
Removed useState